src/daemon/daemon.cpp

```diff
@@ -1233,9 +1239,6 @@ try // clang-format on
             continue;
         }
 
-        if (it->second->current_state() == VirtualMachine::State::running)
-            continue;
-
```

Comment:
Why is this removed? 

It should be essentially a no-op starting an already running VM.

Reply:
I removed it because if there is a start called on a VM that has a delayed shutdown running, then it's supposed to cancel the delayed shutdown.  The way I implemented it in the VM, then the QTimer is never cancelled in this case.  But if I change it as discussed above, then I won't remove this call and instead, will add a call to see if a delayed shutdown is running.

Reply:
But won't you have a different state then? The delayed shutdown state? Not running state?